pkg/util/testing/wrappers.go

```diff
@@ -315,7 +315,6 @@ func MakeFlavor(rf, guaranteed string) *FlavorWrapper {
 		ResourceFlavor: kueue.ResourceFlavorReference(rf),
 		Quota: kueue.Quota{
 			Guaranteed: resource.MustParse(guaranteed),
-			Ceiling:    resource.MustParse(guaranteed),
```

Comment:
hmm, I am surprised that don't have an integration test not assuming ceiling=guaranteed. Should we add one?

Reply:
IIUC, If we don't set the ceiling, the ceiling is equal to generated before. There are also some integration tests, we don't set ceiling --> assuming ceiling=guaranteed?

Reply:
I don't see changes to the integration tests and they are passing, so I assumed no tests are depending on ceiling=guaranteed (meaning can't borrow)

Reply:
The CQs in our integration tests don't belong to a cohort or belong to one of size one. So the ceiling doesn't matter.

We could add a new integration test for cohort, but we can do it in a follow up.

Cohorts are tested at unit level, but it's still good to have one in integration as the multiple creation events could always lead to complications

Reply:
Updated